HeterogeneousCore/ROCmServices/plugins/ROCmService.cc

```diff
@@ -6,7 +6,7 @@
 #include <vector>
 
 #include <hip/hip_runtime.h>
-#include <rocm-core/rocm_version.h>
+#include <rocm_version.h>
```

Comment:
I would suggest
```suggestion
#if HIP_VERSION_MAJOR >= 6
// the location of rocm_version.h changed in HIP/ROCm 6.0
#include <rocm-core/rocm_version.h>
#else
#include <rocm_version.h>
#endif  // HIP_VERSION_MAJOR
```
so we don't have to change it back and forth.

`HIP_VERSION_MAJOR` is defined in `hip/hip_version.h` which is included by `hip/hip_runtime.h`.

Reply:
sounds good